roles/kube-init/templates/kubeadm.cfg.v1.j2

```diff
@@ -0,0 +1,34 @@
+# Full parameters @ https://kubernetes.io/docs/reference/setup-tools/kubeadm/kubeadm-init/
```

Comment:
this file should be `kubeadm.cfg.v1beta.j2`, based in the yaml above? 
Looking https://godoc.org/k8s.io/kubernetes/cmd/kubeadm/app/apis/kubeadm and it seems not to have v1, but v1beta2

Reply:
Yes, except kubelet v1beta. That is why! I backed off to v1beta1 because v1beta2 was broken. 
Anyway it should be fixed in an updated pull request.